macro/src/main/scala/monocle/macros/GenPrism.scala

```diff
@@ -12,12 +13,15 @@ object GenPrism {
 
 private class GenPrismImpl(val c: blackbox.Context) {
   def genPrism_impl[S: c.WeakTypeTag, A: c.WeakTypeTag]: c.Expr[Prism[S, A]] = {
-    import c.universe._
 
-    val (sTpe, aTpe) = (weakTypeOf[S], weakTypeOf[A])
+    c.test(s"constructing Prism with S=${c.weakTypeOf[S]}, A=${c.weakTypeOf[A]}") {
```

Comment:
Oh I see what you mean, so the probably test need to be written in the code of the macro. That's a bit annoying but I understand.

If that's the case, I guess the simplest option would be to keep this code in a separate branch soo that we can run the benchmarks but it will not affect the users. What do you think?